R/mockSummariseLargeScaleCharacteristics.R

```diff
@@ -0,0 +1,43 @@
+#' Function to create mock summarised large-scale characteristics.
+#'
+#' @param seed Seed value for the random mock data generated
+#' @param table_name Name of table from cdm object
+#' @param concept Set of concept ids
+#' @export
+#'
+#' @examples
+#' \donttest{
+#' library(ggoxford)
+#'
+#' mockSummariseLargeScaleCharacteristics()
+#' }
+#'
+mockSummariseLargeScaleCharacteristics <- function(seed=1,table_name="condition_occurrence", concept = c(317009,4266367,378253,1125315,1503328,1516978)) {
```

Comment:
can we use the PatientProfiles functions, similar to what we did in: https://github.com/oxford-pharmacoepi/ggoxford/blob/c5407b96adb27a182c228d233620778646bc2d93/R/mockSummarisedResult.R#L17C1-L55C2